src/components/cards/JufoExpertCard.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useContext } from 'react';
+import CardBase from '../base/CardBase';
+import { Text, Title } from '../Typography';
+import context from '../../context';
+import classes from './JufoExpertCard.module.scss';
+import Button from '../button';
+import { ExpertOverviewModal } from '../Modals/ExpertOverviewModal';
+import EditExpertProfileModal from '../Modals/EditExpertProfileModal';
+
+export const JufoExpertCard: React.FC = () => {
+  const modalContext = useContext(context.Modal);
+  const userContext = useContext(context.User);
+
+  return (
+    <>
+      <CardBase highlightColor="#4E6AE6" className={classes.baseContainer}>
+        <div className={classes.container}>
+          <div className={classes.matchInfoContainer}>
+            <Title size="h4" bold>
+              Willst du Jufo Experte*in werden?
+            </Title>
+            {userContext.user.expertData == null ? (
+              <Text className={classes.emailText} large>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat.
+              </Text>
+            ) : (
+              <Text className={classes.emailText} large>
```

Comment:
In terms of css design we could in my opinion be a bit more consistent with the [SettingsCard](https://github.com/corona-school/web-user-app/blob/fd56bfb7a61d7f81ed08c261f15595e393a45379/src/components/cards/SettingsCard.tsx#L28) on the same page.